scripts/!mods_preload/stronghold_management.nut

```diff
@@ -0,0 +1,550 @@
+local gt = this.getroottable();
+
+gt.Stronghold.setupMainManagementOptions <- function()
+{
+	gt.Stronghold.Main_Management_Options <-
+	[
+		{
+			Text = function(){
+				return format("Upgrade your %s", this.getHome().getSizeName())
+			}
+			ID = "Upgrade",
+			isValid = function(_contract){
+				return _contract.isMainBase() && _contract.getHome().getSize() < 3 && !_contract.getHome().isUpgrading()
```

Comment:
extract this magic number 3 please. `gt.Stronghold.LEVEL_STRONGHOLD` or some such. but not user-exposed settings, somewhere else. e.g. `!config/stronghold_config.nut`

Reply:
Seems unnecessary as the 3 tier structure is firmly established, but I suppose so.